lib/portal

```diff
@@ -13,6 +13,23 @@ function install_mariadb {
 
 # install_portal() - Function that pulls and install the source code of Portal
 function install_portal {
+
+if [[ "$build_image" == "True" ]]; then
+    # Build locally
+    clone_repo portal /opt/portal
+    install_docker
+    docker rm -f ecompdb_portal
+    docker rm -f 1610-1
+    clone_repo ecompsdkos /opt/portal
+    pushd /opt/portal/
+    git clone https://git.onap.org/ui/dmaapbc
```

Comment:
Shouldn't work with:

$ clone_repo "ui/dmaapbc" portal/dmaapbc

or isn't more better to change the clone_repo function to support it?